web/park_api/tests/test_api_v1.py

```diff
@@ -74,7 +75,8 @@ def test_200_city(self):
                         'address': 'An der Frauenkirche 12a\n01067 Dresden\nEinfahrt von Schießgasse\nKontakt: 03 51 / 496 06 03\ntiefgarage-frauenkirche.dresden@gmx.de',
                         'coords': {'lat': 51.051415072, 'lng': 13.7441934672},
                         'forecast': False,
-                        'free': None,
+                        # Don't return free, if not available (to stay backward compatible with ParkAPIv1)
```

Comment:
I'm not sure if some apps may fail if `free`isn't provided, even though the spec says it's not required. OTOH all apps I'm aware of (that are not the website) are unmaintained anyway so there is not much to lose if something breaks.